pack/mts_encounter.json

```
@@ -673,6 +690,8 @@
 		"quantity": 1,
 		"set_code": "tower_defense",
 		"set_position": 9,
+		"text": "The unique rule does not apply to Avenger's Tower. \n <b>Forced Response:</b> After damage is placed here, if there is at least 9[per_player] damage here, remove all of it. Then flip Avenger's Tower over.",
```

I believe when cards have an `A` and `B` side (which this one have as you can see my the card ID in the bottom left), the `A` side should have the `back_text` and the `B` side should have the `text`.

```suggestion
		"back_text": "The unique rule does not apply to Avengers Tower.\n<b>Forced Response:</b> After damage is placed here, if there is at least 9[per_player] damage here, remove all of it. Then flip Avenger's Tower over.",
```